content/en/docs/collector/distributions.md

```diff
@@ -22,3 +24,29 @@ functionality like
 [connectors](../building/connector). The tool used to build distributions
 [ocb](../custom-collector) (OpenTelemetry Collector Builder) is available to
 build your own distributions.
+
+## Vendor Distributions
+
+Some organizations provide a Collector distribution with additional capabilities
+or for improved ease of use. What follows is a list of Collector distributions
+maintained by third parties.
+
+{{% alert title="Note" color="warning" %}} OpenTelemetry **does not validate or
+endorse** the third-party distributions listed in the following table. The list
+is provided as a convenience for the community. {{% /alert %}}
+
+{{% ecosystem/distributions-table filter="vendor-collector" %}}
```

Comment:
Again, third-party is the term we've been using in the registry, etc. WDYT @svrnm?

```suggestion
{{% ecosystem/distributions-table filter="third-party-collector" %}}
```